commands/run.go

```diff
@@ -13,6 +14,7 @@ import (
 func Run(config *proxy.Config) {
 
 	fmt.Println("Ergo Proxy listening on port: " + config.Port)
+	fmt.Println("Ergo Proxy listening for domain: " + config.Domain)
```

Comment:
Could we avoid repeat the "Ergo proxy listening" sentence? How about?

```go
fmt.Println("Ergo Proxy listening on port " + config.Port + " for domains " + config.Domain)
```

Reply:
Good idea.
I pushed the text changes already.